source/release-notes.txt

```diff
@@ -10,6 +10,78 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+|compass| 1.35.0
+----------------
+
+*Released January 11, 2023*
+
+New Features
```

Comment:
[suggestion] Should this be "New Features:"?

Reply:
Updated.